minio/minio
# remove persistence layer for metacache store in memory

## Description
remove persistence layer for metacache store in memory

## Motivation and Context
store the cache in-memory instead of disks to avoid largely
write amplification for listing heavy workloads such as Spark,
Splunk, store in memory instead and let it auto-expire.

## How to test this PR?
Nothing should change from a Listing point of view other
persistence is now in-memory.

## Types of changes

## Checklist:
